### ISSUE TITLE
Few features of the player does not work for Vimeo/Youtube

### ISSUE THREAD

**Issue by ljairath** [CookPete/react-player#560]
Following features did not work for me.

**Vimeo**: Controls, Loop, Fullscreen
**Youtube**: Fullscreen, Controls work except that Youtube icon is displayed.

**Comment by scharissis** [comment]
@ljairath have you tried enabling **modestbranding** to not show the YouTube icon/logo?

```
<ReactPlayer
                      url='https://www.youtube.com/watch?v=mFJZ0HaYYB8'
                      config={{
                        youtube: {
                          playerVars: { modestbranding: 1 }
                        }
                      }}
                    />
```

**Comment by CookPete** [comment]
There's quite a few unspecific problems pointed out here so I'll try and break it down:

> Vimeo controls

The [Vimeo player](https://github.com/vimeo/player.js) has no support for setting controls, so this is not supported. This is [noted in the readme](https://github.com/CookPete/react-player#props) under the `controls` prop.

If you are a Vimeo Plus, PRO, or Business member, you can use [`config.vimeo.playerOptions`](https://github.com/CookPete/react-player#config-prop) to set `background: true` which will ["Enable the player's background mode which hides the controls, autoplays and loops the video"](https://github.com/vimeo/player.js#embed-options).

> Vimeo loop

I assume you are referring to toggling `loop` after a Vimeo video has started playing. This is fixed in `1.9.2`.

>Vimeo fullscreen and Youtube fullscreen

I assume you are referring to the `Fullscreen` function in the demo app on latest Chrome, which was broken due to the Fullscreen API changing. This should now be fixed in the demo app (by upgrading to latest [`screenfull.js`](https://github.com/sindresorhus/screenfull.js)). Other than this, all fullscreen functionality seems to be working correctly.

> Youtube controls work except that Youtube icon is displayed

I'm not sure what the problem is here. With default props (which includes `playerVars: { modestbranding: 1 }` as @scharissis mentioned), the youtube icon only shows when `controls` is _false_ when hovering over the player:

![image](https://user-images.githubusercontent.com/1926029/51800661-5ade4e00-222a-11e9-9b70-114aeb2cc935.png)

When `controls` is _true_, the controls show, but no Youtube icon:

![image](https://user-images.githubusercontent.com/1926029/51800675-85c8a200-222a-11e9-8557-1fc9b200e406.png)

---

If there's anything I've not fixed or explained here that still isn't working as expected. Feel free to open a new issue (or reopen this one) with a clear description of the problem.
